poms/bom/pom.xml

```diff
@@ -6157,16 +6157,16 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-            <dependency>
-                <groupId>com.ibm.mq</groupId>
-                <artifactId>com.ibm.mq.jakarta.client</artifactId>
-                <version>${ibm.mq.client.version}</version>
-            </dependency>
             <dependency>
                 <groupId>com.ibm.icu</groupId>
                 <artifactId>icu4j</artifactId>
                 <version>${icu4j.version}</version>
             </dependency>
```

Comment:
Are those bom additions ibmmq, dropwizard needed in the context of this pr ?

Reply:
I will check it.

Reply:
Removed